src/OpenTelemetry.ClrProfiler.Native/integration_loader.h

```diff
@@ -17,30 +17,21 @@ using json = nlohmann::json;
 
 // LoadIntegrationsFromEnvironment loads integrations from any files specified
 // in the DD_INTEGRATIONS environment variable
-void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const bool isCallTargetEnabled,
-                                     const bool isNetstandardEnabled,
-                                     const std::vector<WSTRING>& disabledIntegrationNames);
+void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const std::vector<WSTRING>& disabledIntegrationNames);
 
 // LoadIntegrationsFromFile loads the integrations from a file
-void LoadIntegrationsFromFile(const WSTRING& file_path, std::vector<IntegrationMethod>& integrationMethods,
-                              const bool isCallTargetEnabled, const bool isNetstandardEnabled,
-                              const std::vector<WSTRING>& disabledIntegrationNames);
+void LoadIntegrationsFromFile(const WSTRING& file_path, std::vector<IntegrationMethod>& integrationMethods, const std::vector<WSTRING>& disabledIntegrationNames);
 
 // LoadIntegrationsFromFile loads the integrations from a stream
-void LoadIntegrationsFromStream(std::istream& stream, std::vector<IntegrationMethod>& integrationMethods,
-                                const bool isCallTargetEnabled, const bool isNetstandardEnabled,
-                                const std::vector<WSTRING>& disabledIntegrationNames);
+void LoadIntegrationsFromStream(std::istream& stream, std::vector<IntegrationMethod>& integrationMethods, const std::vector<WSTRING>& disabledIntegrationNames);
 
 namespace
 {
 
-    void IntegrationFromJson(const json::value_type& src, std::vector<IntegrationMethod>& integrationMethods,
-                             const bool isCallTargetEnabled, const bool isNetstandardEnabled,
-                             const std::vector<WSTRING>& disabledIntegrationNames);
+    void IntegrationFromJson(const json::value_type& src, std::vector<IntegrationMethod>& integrationMethods, const std::vector<WSTRING>& disabledIntegrationNames);
 
     void MethodReplacementFromJson(const json::value_type& src, const WSTRING& integrationName,
-                                   std::vector<IntegrationMethod>& integrationMethods,
-                                   const bool isCallTargetEnabled, const bool isNetstandardEnabled);
+                                   std::vector<IntegrationMethod>& integrationMethods);
 
     MethodReference MethodReferenceFromJson(const json::value_type& src, const bool is_target_method,
```

Comment:
It seems that the usage of this method is now always with the books for `is_target_method` and `is_wrapper_method` either as `true, false` or `false, true` which makes me think that we could live with a single boolean. Anyway, better consider it separate, just a thought that crossed my mind.

Reply:
Good insight Paulo. Yes let's consider this for a later PR